portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,31 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Bubae's Portfolio</h1>
+      <p>This is Bubae's portfolio.</p>
+      <img src="/images/me.jpg"/>
+      <p>I'm Nigerian. I was born on the 23rd of July and I'm a Leo. I'm very patient and tolerant, and I don't get mad often. 
+          I'm 5'5.5" and I'm dark skinned. </p>
+      <p>I have worked on a robotics team at my university. I and four others made an application to aid finaicial
+          literacy in the US. I am on an augmented reality research group currently creating avatars for Mozilla Hubs </p>
+      <p> LinkedIn Profile: https://www.linkedin.com/in/ebubechukwu-bubae-nwafor-ba553a181/ 
```

Comment:
As a suggestion, you could use `<a></a>` to create a [hyperlink](https://w3schools.com/tags/tag_a.asp).